treeherder/etl/tasks/pulse_tasks.py

```diff
@@ -16,7 +17,10 @@ def store_pulse_jobs(job_list, exchange, routing_key):
     newrelic.agent.add_custom_parameter("exchange", exchange)
     newrelic.agent.add_custom_parameter("routing_key", routing_key)
 
-    JobLoader().process_job_list(job_list)
+    try:
+        JobLoader().process_job_list(job_list)
```

Comment:
Yeah, thus far we only get one job at a time.  We had talked about perhaps batching them if performance was a problem, but it hasn't been.  So, yeah, I'm good with changing the names.  I wonder if that would be better in a follow-up PR.

So this checks whether all revisions exist before trying to store any of them.  So if one exists and the other is missing, it'll bail and retry both till all of them exist.